simtools/utils/general.py

```diff
@@ -917,12 +917,13 @@ def get_value_unit_type(value, unit_str=None):
         base_type = extract_type_of_value(base_value)
 
     if unit_str is not None:
-        if base_unit is not None:
```

Comment:
Pycharm complains that lines 902 and 904 are not needed (`base_value = value`, `base_type = ""`) as these arguments are always defined below. I suggest removing them

Reply:
I still see that we can remove `base_type = ""`. i am trying to reproduce locally and see which error you are talking about, but tests are just taking too long

Reply:
If you look carefully, one can see that `base_type` is defined in the function regardless of the outcome of the try and except statement and the if statement, so this first declaration of `base_type=""` should really be removed. Too bad I cant look at it in a reasonable amount of test time (I didnt see a problem when testing general.py alone )

Reply:
OK. Fixed.